src/main/java/io/appform/dropwizard/actors/actor/BaseActor.java

```diff
@@ -100,7 +100,8 @@ protected BaseActor(
     }
 
 
-    abstract protected boolean handle(Message message) throws Exception;
+    abstract protected boolean handle(Message message,
+                                      io.appform.dropwizard.actors.actor.MessageProperties messageProperties) throws Exception;
```

Comment:
Why full package name ?

Reply:
there is already another class with same name in rmq client.

Reply:
Rename this class tom something else to avoid confusion @nottocode

Reply:
renamed.